sources/packages/api/src/types/primaryContact.ts

```diff
@@ -0,0 +1,6 @@
+export interface PrimaryContact {
```

Comment:
Should we keep `AddressInfo` and `PrimaryContact` at this level or move to the entities level?
I would say that we have been creating the types that are related to entities on the entities level. If there are absolutely no modifications at the service level we can reuse them.